priority-queue/PriorityQueueConsumerHigh/PriorityQueueConsumerHighFn.cs

```diff
@@ -0,0 +1,14 @@
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+
+namespace PriorityQueueConsumerHigh
+{
+    public static class PriorityQueueConsumerHighFn
+    {
+        [FunctionName("HighPriorityQueueConsumerFunction")]
+        public static void Run([ServiceBusTrigger("topic_1", "HighPrioritySubscription", Connection = "ServiceBusConnectionString")]string mySbMsg, ILogger log)
```

Comment:
Suggestion: name the topic and the `mySbMsg` param something more "self-descriptive/meaningful" to make the story clearer for folks reading the code.

Reply:
Done, now the name includes the priority of the message

Reply:
Much more descriptive now, thanks.